nikola/plugins/template/jinja.py

```diff
@@ -45,15 +46,35 @@ class JinjaTemplates(TemplateSystem):
     """Support for Jinja2 templates."""
 
     name = "jinja"
-    lookup = None
+    if jinja2 is None:
+        lookup = None
+    else:
+        lookup: Optional[jinja2.Environment] = None
```

Comment:
Consider using a string annotation:

```suggestion
    lookup: 'Optional[jinja2.Environment]' = None
```

Reply:
Could be done and would technically work. But hides the concern and leaves the reader puzzling why the string has been used, as opposed to the plain `jinja2.Environment`. By putting it into the `else` part, the concern is clear and explicit.